Lets Do This/Images.xcassets/listitem-oval.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "listitem-oval@1x.png",
```

Comment:
I think this is just supposed to be named listitem-oval.png without the 1x: https://developer.apple.com/library/mac/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Optimizing/Optimizing.html